README.md

```diff
@@ -26,6 +26,15 @@ Testing on IE11 requires a bit more setup.
 
 Update dependency in package.json to e.g. `"posthog-js": "link:../posthog-js"`, `yarn` and run `yarn build && yarn build-module`
 
+An alternative is to use [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link/). Run `yarn link` in `posthog-js`, and then `yarn link posthog-js` in `posthog`. Once you're done, remember to `yarn unlink`.
+
```

Comment:
This should probably be the default answer, not "an alternative" if it works well?

I use `yalc` personally because `npm link` used to be a hassle. I haven't tried 'yarn link'.

Also "remember to `yarn unlink`" <-- which folder? `posthog` or both?